### PR TITLE
add missing permissions for newly introduced backup/restore cmdlets

### PR DESCRIPTION
**Pull request by dragav**
fix listing of vaults when resource count exceeds 1000  (PowerShell issue 5740)
fix import of certificates from pfx without password (PowerShell issue 5818)

## Checklist

- [X] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [X] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [X] For public API changes to cmdlets:
    - [X] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by dragav** [comment]
The Management SDK was just published (https://www.nuget.org/packages/Microsoft.Azure.Management.KeyVault/2.4.1-alpha) - can an admin trigger another build, please?

**Comment by dragav** [comment]
@schaabs Good catch, fixed. Please have another look.

**Comment by maddieclayton** [comment]
@dragav Looks like you need to rerecord the three failing tests because of the SDK update.

**Comment by maddieclayton** [comment]
on demand here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/715/

File: src/ResourceManager/ServiceFabric/Commands.ServiceFabric.Test/Commands.ServiceFabric.Test.csproj [hunk 1]
```diff
@@ -65,7 +65,7 @@
       <HintPath>..\..\..\packages\Microsoft.Azure.Gallery.2.6.2-preview\lib\net40\Microsoft.Azure.Gallery.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.Azure.KeyVault, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.KeyVault.2.4.0-alpha\lib\net452\Microsoft.Azure.KeyVault.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.Azure.KeyVault.2.3.0-preview\lib\net452\Microsoft.Azure.KeyVault.dll</HintPath>
```
Comment (schaabs) [inline src/ResourceManager/ServiceFabric/Commands.ServiceFabric.Test/Commands.ServiceFabric.Test.csproj]: does this need to point to 3.0.0-alpha to match with the version we reference from Commands.KeyVault?

**Comment by maddieclayton** [comment]
on demand herE: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/719/ (and 720)

**Comment by maddieclayton** [comment]
Test failing unrelated to KV, and this a preview branch, so merging.

**Pull request merged**